server/pulp/server/webservices/views/repositories.py

```diff
@@ -60,37 +56,17 @@ def _merge_related_objects(name, manager, repos):
         else:
             repo_dict[item['repo_id']][name].append(item)
 
-    return repos
 
-
-def _convert_repo_dates_to_strings(repo):
-    """
-    Convert the last_unit_added & last_unit_removed fields of a repository
-    This modifies the repository in place
-
-    :param repo: database representation of a repo
-    :type  repo: dict
+def _process_repos(repo_objs, details, importers, distributors):
     """
+    Serialize a collection of repository objects and add related importers and distributors if
+    requested.
 
-    # convert the native datetime object to a string with timezone specified
-    last_unit_added = repo.get('last_unit_added')
-    if last_unit_added:
-        new_date = dateutils.to_utc_datetime(last_unit_added,
-                                             no_tz_equals_local_tz=False)
-        repo['last_unit_added'] = dateutils.format_iso8601_datetime(new_date)
-    last_unit_removed = repo.get('last_unit_removed')
-    if last_unit_removed:
-        new_date = dateutils.to_utc_datetime(last_unit_removed,
-                                             no_tz_equals_local_tz=False)
-        repo['last_unit_removed'] = dateutils.format_iso8601_datetime(new_date)
-
-
-def _process_repos(repos, details, importers, distributors):
-    """
     Apply standard processing to a collection of repositories being returned to a client. Adds
     the object link and optionally adds related importers and distributors.
 
-    :param repos: collection of repositories
+    :param repo_objs: collection of repository objects
+    :type  repo_objs: list or tuple of model.Repository objects
```

Comment:
I may have a misunderstanding here. I thought it would be better to use `model.Repository` since this module imports `model`. This means that if we were to move where `model` lives, we would not have to update all the doc blocks. I believe that sphinx should know how to create this link. 

Either way, lets add this to our style guide!